packages/calling/src/Metrics/index.ts

```diff
@@ -60,7 +60,7 @@ class MetricManager implements IMetricManager {
           fields: {
             device_url: this.deviceInfo?.device?.clientDeviceUri,
             mobius_url: this.deviceInfo?.device?.uri,
-            calling_sdk_version: VERSION,
+            calling_sdk_version: process.env.CALLING_SDK_VERSION || VERSION,
```

Comment:
Added a fallback here as well, so that even if version fails, we can have some value.

Reply:
cc - @sreenara